core/src/test/java/lionweb/utils/tests/CollectionAssert.java

```diff
@@ -0,0 +1,20 @@
+package lionweb.utils.tests;
+
+import org.junit.Assert;
+
+import javax.annotation.Nonnull;
+import java.util.Iterator;
+import java.util.List;
+
+public class CollectionAssert {
+    public static void AreEqual(@Nonnull List<?> expected, @Nonnull List<?> actual) {
```

Comment:
Not `areEqual`?

Reply:
Yes, it's a bit less Seach+Replace atm. Want to rename it once I copied all the code.